pwndbg/heap/ptmalloc.py

```diff
@@ -5,17 +5,217 @@
 from __future__ import print_function
 from __future__ import unicode_literals
 
+from collections import OrderedDict
+
 import gdb
 
 import pwndbg.events
 import pwndbg.typeinfo
+from pwndbg.constants.ptmalloc import *
+
+
+class Heap(pwndbg.heap.heap.Heap):
```

Comment:
There are currently very few `import *` in the code base, which I'd prefer to keep to a minimum.  Long package names don't bother me so much `pwndbg.heap.heap.Heap`, however I might recommend renaming to e.g. `pwndbg.heap.base.BaseHeap`.